Tests/Functional/Drivers/Queue/StompQueueDriverTest.php

```diff
@@ -20,6 +19,7 @@ class StompQueueDriverTest extends AbstractQueueDriverTest
      * @dataProvider getMessages
      *
      * @param MessageInterface $msg
+     * @group stomp
```

Comment:
I think after the review of the groups you did there are no more groups in the methods

Reply:
Sorry, I forgot to check the Stomp driver also.